### DISCUSSION TITLE
`crates.io` ?

### DISCUSSION THREAD

**Discussion by midnightexigent** [espanso/espanso#1228] [Feature requests and Ideas]
Any reason why this is not on [crates.io](https://crates.io/) ? 

Would allow `cargo install espanso`

**Comment by federico-terzi** [comment]
Hey @midnightexigent,

Thank you for the feedback! At the time espanso was first developed, I've read that cargo wasn't really designed to handle binaries, so other solutions (such as system package managers) should have been preferred. See also https://github.com/rust-cli/team/issues/20
Moreover, with the new version of espanso, the installation command would be significantly more complex than `cargo install espanso`, as there are multiple flags to be passed in order to build it.

That said, I'll leave the issue as a feature request to be investigated in the future.

Cheers :)

**Comment by eclipseo** [comment]
Can this be rediscussed?
We rely on crates.io for packaging script.
